dmgroups.tex

```diff
@@ -128,8 +128,8 @@ \subsection{DM Change Control Board \label{sect:dmccb}}
 	\end{itemize}
 \item Membership
 	\begin{itemize}
-	\item Chaired by the Systems Engineer
-	\item Members include the DM Software Architect (\S\ref{role:softarc}), DM Operations Architect (\S\ref{role:opsarc}), DM System Interfaces Scientist (\S\ref{role:dmis}), DM SQuaRE T/CAM (\S\ref{role:tcam}), DM Release Manager (\S\ref{role:dmrm}) and DM Project Manager (\S\ref{role:dmpm})
+	\item Chaired by the Systems Engineer (\secref{role:sysengineer}).
+	\item The DMCCB has the same membership as the Systems Engineering Team (\secref{sect:sysengt}).
```

Comment:
If the CCB and the SET have the same membership, is there any real need to call them out as separate groups? Perhaps just "the Systems Engineering Team also serves as the change control board for DM, and as such its responsibilities include..."?